packages/api/src/middlewares/auth/index.ts

```diff
@@ -79,4 +81,30 @@ function authPermissionMiddleware(permission?: PermissionRoles[]) {
   };
 }
 
-export { authMiddleware, authPermissionMiddleware };
+function authPointsMiddleware(
+  req: IAuthRequest,
+  res: Response,
+  next: NextFunction,
+) {
+  try {
+    const authorization = req?.headers?.authorization;
```

Comment:
verify the if origin of request is bako-points ui

add new var on process.env with this url